packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.test.ts

```diff
@@ -77,6 +84,16 @@ describe('Overrides Resolver Loader', () => {
             },
             expects: (output: any) => {
                 expect(output.modules[1].source).toBe('// Base Project - Sample Page')
+                // Verify stats
+                const stats = output.overrideStats
+                expect(stats).toHaveLength(1)
+                expect(stats[0]).toMatchObject({
+                    sourceExtension: '@salesforce/extension-this',
+                    original: expect.stringContaining('extension-this/src/pages/sample-page'),
+                    resolved: expect.stringContaining(
+                        'app/overrides/@salesforce/extension-this/pages/sample-page'
+                    )
+                })
```

Comment:
I've never seen an expect inside of an inspect like this before. You are doing that because the paths are probably starting at the root of the file system ?